pyFV3/__init__.py

```diff
@@ -1,8 +1,15 @@
 from ._config import DynamicalCoreConfig
 from .dycore_state import DycoreState
-from .stencils.fv_dynamics import DynamicalCore
-from .stencils.fv_subgridz import DryConvectiveAdjustment
-from .wrappers.geos_wrapper import GeosDycoreWrapper
+from .stencils import DryConvectiveAdjustment, DynamicalCore
+from .wrappers import GeosDycoreWrapper
```

Comment:
I'm not totally convinced we want to expose the wrappers here, but it's probably ok with just the GEOS wrapper, though I may want to revisit if it starts getting unwieldy

Reply:
Fair opposition.

Technically a Wrapper is so bespoke it's a bit of a stretch for me to call it a "public" API. I wouldn't object to removing it.

I might in the future take it off `pyFV3` entirely and move it to GEOS itself